firefox_media_tests/playback/test_ultra_low_bandwidth.py

```diff
@@ -0,0 +1,16 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from marionette import BrowserMobProxyTestCaseMixin
+
+from media_utils.video_puppeteer import VideoPuppeteer
+from media_test_harness.testcase import NetworkBandwidthTestCase
+
+
+class TestPlaybackLimitingBandwidth(NetworkBandwidthTestCase,
```

Comment:
This class has the same name as the other in `test_playback_limiting_bandwidth.py`. Is that intentional? I'm not sure whether it's a problem...